### PR TITLE
CU-86a1j4qjt - NEON2 - Implement Sentry or similar tool to track erro…

### PR DESCRIPTION
**Pull request by LeonardoDizConde**
…rs in productionCU-86a1j4qjt - NEON2 - Implement Sentry or similar tool to track errors in production

**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-86a1j4qjt NEON2 - Implement Sentry or similar tool to track errors in production](https://app.clickup.com/t/86a1j4qjt)

File: config/sentry.config.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const sentryConfig = {
+  attachStacktrace: true,
+  autoSessionTracking: true,
+  debug: false,
+  enableTracing: true,
+  environment: 'NEON-2',
+  release: version,
+  // Capture Replay for 10% of all sessions,
+  // plus for 100% of sessions with an error
```
Comment (raulduartep) [inline config/sentry.config.js]: Remove these comments

File: app/util/SentryReactHelper.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import * as ElectronRedererSentry from '@sentry/electron/renderer'
+import * as ReactSentry from '@sentry/react'
+import { isSentryEnable } from '../../config/sentry.config'
+
+const { sentryConfig } = require('../../config/sentry.config')
+
+const isProductionMode = Boolean(true)
```
Comment (raulduartep) [inline app/util/SentryReactHelper.js]: Is it right ? Should it always be production ?

File: config/sentry.config.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  debug: false,
+  enableTracing: true,
+  environment: 'NEON-2',
+  release: version,
+  replaysOnErrorSampleRate: 1.0,
+  replaysSessionSampleRate: 0.1,
+  // Set `tracePropagationTargets` to control for which URLs distributed tracing should be enabled
+  tracePropagationTargets: ['localhost'],
+  // Set tracesSampleRate to 1.0 to capture 100%
+  // of transactions for performance monitoring.
```
Comment (raulduartep) [inline config/sentry.config.js]: Remove these comments too

File: config/sentry.config.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+const sentryConfig = {
+  attachStacktrace: true,
+  autoSessionTracking: true,
+  debug: false,
+  enableTracing: true,
+  environment: 'NEON-2',
+  release: version,
+  replaysOnErrorSampleRate: 1.0,
+  replaysSessionSampleRate: 0.1,
+  // Set `tracePropagationTargets` to control for which URLs distributed tracing should be enabled
```
Comment (raulduartep) [inline config/sentry.config.js]: Remove these comments too

**Review by raulduartep — APPROVED** [review]

**Pull request merged**